conf/application.conf

```diff
@@ -172,6 +172,13 @@ mongo-async-driver {
   }
 }
 
+today-date-for-validation-flag {
```

Comment:
as this is not a flag really , can we rename this to something like `retrospective-declaration-date`

Reply:
why is not a flag? - contains a boolean and will need to be removed on 1/1/21